FashionConnect_ui/src/components/Main/Main.jsx

```diff
@@ -1,12 +1,25 @@
 import "./Main.css"
 import NavBar from "../NavBar/NavBar";
+import { useState ,useEffect } from "react";
 import Products from "../Products/Products";
 
 export default function Main() {
+    const [products, setProducts] = useState([]);
+
+    useEffect(() => {
+        const fetchProducts = async () => {
+        const response = await fetch('http://localhost:3000/products');
```

Comment:
You can create a constant of this URL in your file for reference in other places if needed.

Reply:
Ok